Nos/Views/Discover/FeaturedAuthorsView.swift

```diff
@@ -131,4 +135,22 @@ struct FeaturedAuthorsView: View {
         }
         .background(Color.profileBgTop)
     }
+
+    private func findOrCreateAuthors() {
+        for featuredAuthorNpub in FeaturedAuthorCategory.all.npubs {
+            do {
+                guard let publicKey = PublicKey(npub: featuredAuthorNpub) else {
+                    Log.error(
+                        "Could create public key for npub: \(featuredAuthorNpub)" +
+                        "The npub in FeaturedAuthorCategory may be invalid."
+                    )
+                    return
```

Comment:
Should we change this `return` to `continue` instead? Just wondering. The `continue` would allow us to create the remaining authors.

Reply:
You're absolutely right. In dev, we probably want this to just fail immediately (maybe even with an assert, which I guess I could add). In prod, though, we probably want to just keep going. I've updated this to use an assert and continue.

Reply:
Fixed!